File: CPP/Program/Sumisang.cpp
#include <iostream>
#include <string>

using namespace std;

// Kelas Sumisang
class Sumisang{
    private:
        string code;
        string name;
        int price;

    public:
        // Konstruktor default dan parameterized
        Sumisang(){
            this->code = "";
            this->name = "";
            this->price = 0;
        }

        Sumisang(string code, string name, int price){
            this->code = code;
            this->name = name;
            this->price = price;
        }
        
        // Setter dan Getter untuk code
        void setCode(string code){
            this->code = code;
        }

        string getCode(){
            return this->code;
        }

        // Setter dan Getter untuk name
        void setName(string name){
            this->name = name;
        }

        string getName(){
            return this->name;
        }

        // Setter dan Getter untuk price
        void setPrice(int price){
            this->price = price;
        }

        int getPrice(){
            return this->price;
        }

        ~Sumisang(){
        }
};

File: Java/Program/Sumisang.java
// kelas Sumisang
public class Sumisang {
    // atribut
    private String code;
    private String name;
    private int price;

    // konstruktor default
    public Sumisang() {
        this.code = "";
        this.name = "";
        this.price = 0;
    }

    // konstruktor parameterized
    public Sumisang(String code, String name, int price) {
        this.code = code;
        this.name = name;
        this.price = price;
    }

    // setter dan getter untuk code
    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    // setter dan getter untuk name
    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    // setter dan getter untuk price
    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }
}
